src/draive/conversation/realtime/default.py

```
@@ -1,210 +1,156 @@
-                    )
-                )
-                await memory.remember(element)  # add to memory
-
-            input_queue.finish()
-            ctx.log_debug("...input processing finished!")
-
-        except CancelledError:
-            # just finish input on cancel
-            input_queue.finish()
+    async def open_session() -> RealtimeConversationSession:  # noqa: C901
+        session: LMMSession = await session_scope.__aenter__()
```

Pylint (3.3.7)

[convention] Unnecessarily calls dunder method __aenter__. Invoke context manager directly.

(C2801)